storage/src/rocksdb_client.cpp

```diff
@@ -401,10 +401,13 @@ KeyValuePair ClientIterator::seekAtLeast(const Sliver &_searchKey) {
  * @return The previous key value pair.
  */
 KeyValuePair ClientIterator::previous() {
+  if (!m_iter->Valid()) {
+    LOG_ERROR(logger, "Iterator is not valid");
+    return KeyValuePair();
+  }
   m_iter->Prev();
-
   if (!m_iter->Valid()) {
-    LOG_ERROR(logger, "Iterator out of bounds");
+    LOG_ERROR(logger, "No previous key");
```

Comment:
Nit: I think this should perhaps be a warning, since it's reasonable to ask previous on a valid iterator, unlike the case above.

Reply:
Agreed, changing as you suggest. Copy-paste...